test/e2e-go/features/transactions/asset_test.go

```diff
@@ -143,15 +143,31 @@ func TestAssetValidRounds(t *testing.T) {
 	firstValid = 0
```

Comment:
Not related to this PR, but did you mean to drop `err` above (line 140)?

Reply:
good catch I'll fix

Reply:
good catch I'll fix